### DISCUSSION TITLE
PF4 DatePicker DateFormat and DateParser

### DISCUSSION THREAD

**Discussion by steverhoades** [data-driven-forms/react-forms#1329] [General]
Anyone know how to control the format of the date picker for PF4 such that it's MM-DD-YYYY?

**Comment by Hyperkid123** [comment]
Hi. You check the PF docs: https://www.patternfly.org/v4/components/date-picker#calendarformat

If you don't find the answer you are looking for, I recommend opening an issue/discussion in the PF repo: https://github.com/patternfly/patternfly-react

**Comment by steverhoades** [comment]
I've checked the PF4 docs and the component itself does support this. Passing the props via data-driven-forms doesn't seem to work as the props never land on the PF4 component. I was hoping I was missing something obvious, I'll dig a little deeper today.

**Comment by steverhoades** [comment]
Took a quick look at the MUI data-driven-forms code and it passes a `DatePickerProps` prop to the MUI date picker component. 
https://github.com/data-driven-forms/react-forms/blob/master/packages/mui-component-mapper/src/date-picker/date-picker.js#L66

My guess is there is something missing on the PF4 wrapper side. If that's the case I'll provide a PR as soon as possible. Cheers!

**Reply by Hyperkid123** [reply]
All props should land in the PF4 component via [the rest spread](https://github.com/data-driven-forms/react-forms/blob/master/packages/pf4-component-mapper/src/date-picker/date-picker.js#L22).

**Comment by steverhoades** [comment]
That was my mistake. You are correct! I had placed those props in the `FieldProps` attribute in my schema. Adding them next to FieldProps, label etc.. works perfect. Much appreciated!
